cmd/sourced/dir/dir.go

```diff
@@ -47,3 +48,8 @@ func DownloadURL(url, dst string) error {
 	_, err = io.Copy(out, resp.Body)
 	return err
 }
+
+// TmpPath returns the absolute path for /tmp/sourced
```

Comment:
comments says `/tmp/sourced` but the code `/tmp/srcd`

Reply:
I was going to propose this change later in `tmp` and `~`, got it mixed here.
Fixed in 34cfe44